scripts/product.sql

```diff
@@ -354,7 +351,8 @@ ALTER TABLE `catalog_product_entity_media_gallery_value_to_entity`
     DROP FOREIGN KEY `CAT_PRD_ENTT_MDA_GLR_VAL_TO_ENTT_ROW_ID_CAT_PRD_ENTT_ROW_ID`,
     DROP INDEX `CAT_PRD_ENTT_MDA_GLR_VAL_TO_ENTT_ROW_ID_CAT_PRD_ENTT_ROW_ID`,
     ADD CONSTRAINT `CAT_PRD_ENTT_MDA_GLR_VAL_TO_ENTT_VAL_ID_ENTT_ID` UNIQUE KEY (`value_id`,`entity_id`),
-    DROP PRIMARY KEY, ADD PRIMARY KEY (`value_id`),
+    DROP KEY CAT_PRD_ENTT_MDA_GLR_VAL_TO_ENTT_VAL_ID_ROW_ID,
+    ADD PRIMARY KEY (`value_id`,`entity_id`),
```

Comment:
![module-catalog-staging](https://github.com/user-attachments/assets/65049015-71ff-4ad7-a58a-cad3078a3ad3)

The module catalog staging of EE adds primary key with a specific name CAT_PRD_ENTT_MDA_GLR_VAL_TO_ENTT_VAL_ID_ROW_ID (the simple `DROP PRIMARY KEY` statemente does not work). The Open Source has a primary key on value_id and entity_id